modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/dashboard/pages/SettingsModals/WhatsNewSettings.tsx

```diff
@@ -0,0 +1,284 @@
+		{
+			label: '5:30',
+			value: '5:30',
+		},
+		{
+			label: '6:00',
+			value: '6:00',
+		},
+	];
+
+	const daysOfTheWeekButtons: cardButtonType[] = [
+		{
+			active: false,
+			name: 'sun',
+		},
+		{
+			active: false,
+			name: 'mon',
+		},
+		{
+			active: false,
+			name: 'tue',
+		},
+		{
+			active: false,
+			name: 'wed',
+		},
+		{
+			active: false,
+			name: 'thu',
+		},
+		{
+			active: false,
+			name: 'fri',
+		},
+		{
+			active: false,
+			name: 'sat',
+		},
+	];
+
+	const periodOfTheDay: cardButtonType[] = [
+		{
+			active: false,
+			name: 'am',
+		},
+		{
+			active: false,
+			name: 'pm',
+		},
+	];
+
+	const [buttons, setButtons] = useState<cardButtonType[]>(
+		daysOfTheWeekButtons
+	);
+	const [periodButtons, setPeriodButtons] = useState<cardButtonType[]>(
+		periodOfTheDay
+	);
+
+	const selectCardDaysOfWeek = (currentIndex: number) => {
+		const newButtons = buttons.map((button, index) => {
+			if (currentIndex === index) {
+				return {
+					...button,
+					active: !button.active,
+				};
+			}
+
+			return button;
+		});
+
+		setButtons(newButtons);
+	};
+
+	const selectPeriodCard = (currentIndex: number) => {
+		periodOfTheDay[currentIndex].active = !periodOfTheDay[currentIndex]
+			.active;
+
+		setPeriodButtons(periodOfTheDay);
+	};
+
+	const ScheduleUpdates = () => (
+		<div className="dashboard-whats-new-modal-container mt-3 pb-2 px-5">
+			<div className="font-weight-bolder"> Schedule</div>
+
+			<div className="font-weight-semi-bold mb-3 mt-3">
+				Generate list on
+			</div>
+
+			<div className="d-flex dashboard-whats-new-modal-buttons flex-row">
+				{buttons.map((button, index) => (
+					<div key={index}>
+						<ClayButton
+							className={classNames(
+								'mr-4 px-2 text-paragraph-sm text-uppercase dashboard-whats-new-modal-period-buttons',
+								{
+									'dashboard-whats-new-modal-active-buttons':
+										button.active,
+								}
+							)}
+							displayType="secondary"
+							onClick={() => {
+								selectCardDaysOfWeek(index);
+							}}
+						>
+							{button.name}
+						</ClayButton>
+					</div>
+				))}
+			</div>
+
+			<div className="font-weight-semi-bold mt-4">Data from</div>
+
+			<div className="d-flex dashboard-whats-new-modal-buttons dashboard-whats-new-modal-lines mt-3">
+				<ClaySelect
+					aria-label="Select Label"
+					className="dashboard-whats-new-modal-period-options mr-3"
+					id="mySelectId"
+				>
+					{dataFromOptions.map((item, index) => (
+						<ClaySelect.Option
+							key={index}
+							label={item.label}
+							value={item.value}
+						/>
+					))}
+				</ClaySelect>
+
+				<ClaySelect
+					aria-label="Select Label"
+					className="dashboard-whats-new-modal-options mr-3"
+					id="mySelectId"
+				>
+					{dataHoursOptions.map((item, index) => (
+						<ClaySelect.Option
+							key={index}
+							label={item.label}
+							value={item.value}
+						/>
+					))}
+				</ClaySelect>
+
+				{periodButtons.map((period, index) => {
```

Comment:
Please change the parameter name here and put the parameter types

```suggestion
				{periodButtons.map((periodButton:cardButtonType, index:number) => {
```

Reply:
Done.